scripts/docker/buildpack-deps/Dockerfile.ubuntu1804

```diff
@@ -92,7 +92,7 @@ RUN set -ex; \
 ARG HERA_HASH="622663cb3bc1fa07213c6e1fe8070c5c259096e75039a46d014b2a3448b5cf44"
```

Comment:
need to change this.

Reply:
Yep.. we still need to wait to get the hashes.. I just prepared this already

Reply:
Ah, yes! you're right!

Reply:
Hmm.. the script is currently not verifying `EVMONE_HASH`/`HERA_HASH`